rpc_jobs/rpc_aio.yml

```diff
@@ -211,15 +245,18 @@
                 kibana_branch = env.UPGRADE_FROM_REF ?: env.RPC_BRANCH
                 kibana.kibana(kibana_branch)
                 holland.holland()
-                if (env.STAGES.contains("Upgrade")) {{
-                  deploy.upgrade(environment_vars: environment_vars)
+                if (env.STAGES.contains("Major Upgrade")) {{
+                  deploy.upgrade_major(environment_vars: environment_vars)
                   deploy.addChecksumRule()
                   maas.deploy()
                   maas.verify()
                   tempest.tempest()
                   kibana.kibana(env.RPC_BRANCH)
                   holland.holland()
                 }}
+                if (env.STAGES.contains("Leap Upgrade")) {{
```

Comment:
Agreed. At some point we'll maybe even be able to re-use existing upgrade stages.